nomenclature/definitions/variable/energy/energy-secondary.yaml

```diff
@@ -127,3 +131,166 @@ Secondary Energy|Solids:
    description: Solid secondary energy carriers (e.g., briquettes, coke, wood
                 chips, wood pellets)
    unit: EJ/yr
+   
+Secondary Energy|Transportation:
+   description: Total production of transportation
+   unit: Gvkm/yr
+
+Secondary Energy|Transportation|Freight:
```

Comment:
Please also use the `<Transport>` replacement here (or define a separate replacement if the terms are different).

Reply:
I removed Transportation from `Secondary Energy` and introduced `Energy Services|Transportation` as suggested above